src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -169,6 +174,11 @@ export class InputDatePicker
     }
   }
 
+  /**
+   * @internal
+   */
+  @Prop({ mutable: true, reflect: true }) open = false;
```

Comment:
@jcfranco `open` doesn't exist on `input-date-picker` yet. I added this prop for now.

Reply:
created: https://github.com/Esri/calcite-components/pull/4972

Reply:
Awesome, thanks!